core/routemgmt/getApi.js

```diff
@@ -119,11 +119,11 @@ function readApiDocument(cloudantDb, docId, params) {
   return new Promise (function(resolve,reject) {
     cloudantDb.get(docId, params, function(error, response) {
       if (!error) {
-        console.log('success', JSON.stringify(response));
+        console.log('DB success', JSON.stringify(response));
         resolve(response);
       } else {
-        console.error("DB error: ",error.name, error.error, error.reason, error.headers.statusCode);
-        if (error.headers.statusCode == 404) {  // When NOT FOUND, return empty results
+        console.error('DB error: '+JSON.stringify(error))
+        if (error.headers && error.headers.statusCode == 404) {  // When NOT FOUND, return empty results
```

Comment:
read `statusCode` from error object instead and also use `===` for consistency and jshint.
```
if(error.statusCode === 404) {...
```

For reference here is how cloudant/nano return error 
https://github.com/dscape/nano/blob/master/lib/nano.js#L244